src/jpdb.rs

```diff
@@ -190,6 +197,11 @@ impl VocabCard<'_> {
         service: &mut BufferedService,
         sentence: &str,
     ) -> Result<()> {
+        info!("custom sen:\n{}", sentence);
```

Comment:
no point abbreviating that, imo

Reply:
Changed to 
`debug!("custom sentence: {}", sentence);`
May have no need for it, though.